EIPS/eip-7425.md

```diff
@@ -0,0 +1,177 @@
+---
+eip: 7425
+title: Tokenized Reserve
+description: Reserve with user auditing
```

Comment:
Could do with a bit more description. Maybe think about what this proposal brings over the practice that exists before EIP-7425?